### PR TITLE
[Newsletters][LPN-676] add confirmation email on legislatives newsletter subscription

### PR DESCRIPTION
**Pull request by elvino29**
```
INSERT INTO cms_block
(name, description, content)
VALUES('legislative-newsletter-thanks-content', 'Texte affiché pour la page de confirmation de souscription à la newsletter d''un candidat aux législatives', '<h1 class="text--large">
    Votre inscription à notre newsletter est validée. Merci.
</h1>');
```

### CODE REVIEW

File: src/Entity/LegislativeNewsletterSubscription.php [hunk 1]
```diff
@@ -105,9 +105,20 @@ class LegislativeNewsletterSubscription implements RecaptchaChallengeInterface
...
 
+    /**
+     * @ORM\Column(type="uuid", unique=true, nullable=true)
+     */
+    private UuidInterface $token;
+
+    /**
+     * @ORM\Column(type="datetime", nullable=true)
+     */
+    private ?\DateTimeInterface $confirmedAt;
```
Comment (ottaviano) [inline src/Entity/LegislativeNewsletterSubscription.php:116]:
```suggestion
    private ?\DateTimeInterface $confirmedAt = null;
```

File: src/Legislative/Newsletter/Listener/SendValidationMailSubscriber.php [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+    public static function getSubscribedEvents()
+    {
+        return [Events::NEWSLETTER_SUBSCRIBE => 'sendValidationEmail'];
+    }
+
+    public function sendValidationEmail(LegislativeNewsletterEvent $event): void
+    {
+        $subscription = $event->getNewsletter();
+        if (null === $subscription->getUuid() || null === $subscription->getToken()) {
```
Comment (ottaviano) [inline src/Legislative/Newsletter/Listener/SendValidationMailSubscriber.php:37]: When can this happen?

File: templates/legislatives/newsletter/thanks.html.twig [hunk 3]
```diff
@@ -0,0 +1,22 @@
+{% extends 'legislatives/newsletter/_layout.html.twig' %}
+
+{% block page_title 'Vous êtes désormais inscrit à la newsletter ! - Legislatives' %}
+
+{% block legislatives_content %}
+    <main class="legislatives_newsletter__content">
+        <section class="legislatives_newsletter__content__form l__wrapper l__row l__row--top l__row--h-stretch-around l__tablet--col space--40-0">
+            <div class="l__wrapper">
+                <h2 class="text--large b__nudge--bottom-larger">
+                    Vous venez de vous inscrire à la newsletter d'un candidats aux élection legislatives. Merci !
```
Comment (ottaviano) [inline templates/legislatives/newsletter/thanks.html.twig:10]:
```suggestion
                    Vous venez de vous inscrire à la newsletter d'un candidat aux élections legislatives. Merci !
```

File: templates/legislatives/newsletter/thanks.html.twig [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+                </h2>
+
+                <p class="text--body text--dark font-roboto b__nudge--bottom-larger">
+                    Vous recevrez désormais régulièrement des informations sur la campagne du candidat (événements, projets…).<br /><br />
+
+                    À bientôt,<br />
+                    L'équipe de La République En Marche !
```
Comment (ottaviano) [inline templates/legislatives/newsletter/thanks.html.twig:17]: is it valid content?

File: src/Mailer/Message/Legislatives/AbstractLegislativeNewsletterMessage.php [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+
+namespace App\Mailer\Message\Legislatives;
+
+use App\Mailer\Message\Message;
+
+abstract class AbstractLegislativeNewsletterMessage extends Message
+{
+    protected static function updateSenderInfo(Message $message): Message
+    {
+        $message->setSenderEmail('contact@avecvous.fr');
```
Comment (ottaviano) [inline src/Mailer/Message/Legislatives/AbstractLegislativeNewsletterMessage.php:11]: is it ok for this email and not `ne-pas-repondre@`?

File: src/Legislative/Newsletter/Events.php [hunk 6]
```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace App\Legislative\Newsletter;
+
+final class Events
+{
+    public const NEWSLETTER_SUBSCRIBE = 'newsletter.subscribe';
+    public const NEWSLETTER_CONFIRMATION = 'newsletter.confirmation';
+    public const NEWSLETTER_UNSUBSCRIBE = 'newsletter.unsubscribe';
```
Comment (ottaviano) [inline src/Legislative/Newsletter/Events.php:9]: where is used?

File: templates/legislatives/newsletter/thanks.html.twig (hunk 3 above)
Comment (elvino29) [reply templates/legislatives/newsletter/thanks.html.twig:10]: I'm waiting for Laïla to give me the text to put on this page. What I put here is just to test the good working of this page

File: templates/legislatives/newsletter/thanks.html.twig (hunk 4 above)
Comment (elvino29) [reply templates/legislatives/newsletter/thanks.html.twig:17]: no
waiting for the good one

**Review by ottaviano — APPROVED** [review]

**Pull request merged**